app/models/planning_application.rb

```diff
@@ -47,6 +47,9 @@ class WithdrawOrCancelError < RuntimeError; end
     has_many :requests, class_name: "ValidationRequest"
     has_many :assessment_details, -> { by_created_at_desc }, inverse_of: :planning_application
     has_many :permitted_development_rights, -> { order :created_at }, inverse_of: :planning_application
+    has_many :planning_application_constraints
```

Comment:
We think that according to the original diagram this should be

```
has_many :planning_application_constraint_queries
```

However, we also think that we're going to need to support custom constraints that aren't associated with a query. So in fact we'll want:

```suggestion
    has_many :planning_application_constraint_queries # for constraints from planx
    has_many :planning_application_constraints # for manually-added constraints
```

Reply:
`has_many :planning_application_constraints_queries`